build_scripts/update_redis_server.py

```diff
@@ -13,11 +13,19 @@
 if __name__ == "__main__":
     shutil.rmtree('redis.submodule')
     with tempfile.TemporaryDirectory() as tempdir:
+        print(f'Downloading {url} to temp directory {tempdir}')
         ftpstream = urllib.request.urlopen(url)
         tf = tarfile.open(fileobj=ftpstream, mode="r|gz")
         directory = tf.next().name
-        print(directory)
+        
+        print('Extracting archive {directory}')
```

Comment:
I am not sure if this will work in here. Maybe it's typo and you wanna use format string?
If it's format string, then should we make python3 is required and exclude python2?

Reply:
This utility is for a developer to run from the command line as a helper to update the redis source in the redis.submodule directory.  

It's not included in the package or run in the ci/cd pipeline.

Reply:
Good catch